pkg/cmd/util/output/backup_describer.go

```diff
@@ -243,6 +254,7 @@ func DescribeBackupStatus(d *Describer, backup *velerov1api.Backup, details bool
 		d.Println()
 	}
 
+	// TODO(nrb-csi): Should CSI snapshots increment VolumeSnapshotsAttempted? If so, we'll need a way to differentiate here.
 	if status.VolumeSnapshotsAttempted > 0 {
 		if !details {
 			d.Printf("Persistent Volumes:\t%d of %d snapshots completed successfully (specify --details for more information)\n", status.VolumeSnapshotsCompleted, status.VolumeSnapshotsAttempted)
```

Comment:
we probably want to try to disambiguate here between non-CSI and CSI snapshots - maybe instead of "Persistent Volumes:" we say "Velero-Native Snapshots" or "non-CSI Snapshots" or something like that

Reply:
(note this applies to various lines in this section, not just this one)

Reply:
Yeah, I should probably comb through here and revise the wording every where. I did it in the case that there were 0 velero-native snapshots, but nowhere else.